protocol/daemons/pricefeed/client/client.go

```diff
@@ -117,7 +117,7 @@ func (c *Client) start(ctx context.Context,
 	appFlags appflags.Flags,
 	logger log.Logger,
 	grpcClient daemontypes.GrpcClient,
-	exchangeIdToStartupConfig map[types.ExchangeId]*types.ExchangeStartupConfig,
+	exchangeIdToStartupConfig map[types.ExchangeId]*types.ExchangeQueryConfig,
```

Comment:
type rename: `ExchangeStartupConfig` -> `ExchangeQueryConfig`

Reply:
Probably should rename variable `exchangeIdToStartupConfig` accordingly?

Reply:
Ok, renamed everywhere!